2019/5xxx/CVE-2019-5624.json

```diff
@@ -1,18 +1,127 @@
 {
-    "CVE_data_meta": {
-        "ASSIGNER": "cve@mitre.org",
-        "ID": "CVE-2019-5624",
-        "STATE": "RESERVED"
-    },
-    "data_format": "MITRE",
-    "data_type": "CVE",
-    "data_version": "4.0",
-    "description": {
-        "description_data": [
-            {
-                "lang": "eng",
-                "value": "** RESERVED ** This candidate has been reserved by an organization or individual that will use it when announcing a new security problem. When the candidate has been publicized, the details for this candidate will be provided."
-            }
+  "data_type": "CVE",
+  "data_format": "MITRE",
+  "data_version": "4.0",
+  "generator": {
+    "engine": "Vulnogram 0.0.6"
+  },
+  "CVE_data_meta": {
+    "ID": "CVE-2019-5624",
+    "ASSIGNER": "cve@rapid7.com",
+    "DATE_PUBLIC": "2019-04-24T18:00:00.000Z",
+    "TITLE": "Rapid7 Metasploit Framework Zip Import Directory Traversal",
+    "AKA": "",
+    "STATE": "PUBLIC"
+  },
+  "source": {
+    "defect": [
+      "MS-3923"
+    ],
+    "advisory": "",
+    "discovery": "UNKNOWN"
+  },
+  "affects": {
+    "vendor": {
+      "vendor_data": [
+        {
+          "vendor_name": "Rapid7",
+          "product": {
+            "product_data": [
+              {
+                "product_name": "Metasploit Framework",
+                "version": {
+                  "version_data": [
+                    {
+                      "version_name": "4.14.0",
+                      "version_affected": "<=",
+                      "version_value": "4.14.0",
+                      "platform": ""
+                    }
+                  ]
+                }
+              }
+            ]
+          }
+        }
+      ]
+    }
+  },
+  "problemtype": {
+    "problemtype_data": [
+      {
+        "description": [
+          {
+            "lang": "eng",
+            "value": "CWE-22"
+          },
+          {
+            "lang": "eng",
+            "value": "Improper Limitation of a Pathname to a Restricted Directory ('Path Traversal')"
+          }
         ]
+      }
+    ]
+  },
+  "description": {
+    "description_data": [
+      {
+        "lang": "eng",
+        "value": "Rapid7 Metasploit Framework suffers from an instance of CWE-22, Improper Limitation of a Pathname to a Restricted Directory ('Path Traversal') in the Zip import function of Metasploit. Exploiting this vulnerability can allow an attacker to execute arbitrary code in Metasploit at the privilege level of the user running Metasploit."
+      },
+      {
+        "lang": "eng",
+        "value": "This issue affects:\nRapid7 Metasploit Framework version 4.14.0 and prior versions."
```

Comment:
I don't know how downstream parsers will handle description being split into two fields. In previous JSON submissions I think we've included the version info in a single description field, so probably safest to move line 73 content into line 69.

Reply:
Sure thing. This is straight from Vulnogram, though. I can smoosh it all together, though, shouldn't be an issue.